docs/README.md

```diff
@@ -1,7 +1,198 @@
 # Return App V3 <sup>BETA</sup>
 
-Return app v3 is still in BETA. Docs are WIP.
+Return app v3 is still in BETA. v2 will no longer be supported so please do not install it. 
+ Docs are WIP.
 
+## Description
+
+The **Return App** gives merchants the ability to manage the Return Request Process on their store. 
+
+## Features
+
+### Admin Return Request List
+In this section of the merchant's admin, merchants are capable of visualizing and managing all the return requests created by their customers. 
+### Admin Return Settings
+In this section of the merchant's admin, merchants control what are the conditions of allowing the return process for a given item or items per customer. 
+- **Max Days**: when an order creation date is older than the Max Days to return, user won't be able to select that order for a return
+- **Terms and Conditions**: link to the Terms and conditions offered by the store. (not provided by the app)
+- **Excluded categories**: List of categories to be excluded for the return process. Any item that belongs to any of these categories, will not be allowed to be returned.
+- **Return Payment Options**: 
+
+  -- Same as Order: Payment will be refunded to the same payment method used in the order. 
+
+  -- Allowed to choose: customer will select any of the refund options selected by the store. 
+      **Disclaimer**: Only Credit Card triggers the automatic refund. **WIP: automatic refund for any payment method. **
+- **Custom Return Reasons**: Allows the store to define their own custom return reasons. This setting will overwrite the default reasons. 
+The custom Return Reasons can be translated manually by the admin. 
+    |Default return reason|
+    |---------|
+    |Accidental Order|
+    |Better Price|
+    |Performance|
+    |Incompatible|
+    |Item Damaged|
+    |Missed Delivery|
+    |Missing Parts|
+    |Box Damaged|
+    |Different Product|
+    |Defective|
+    |Arrived in addition|
+    |No Longer Needed|
+    |Unauthorized purchase|
+    |Different from website|
+
+- **Other Option**: toggle this to include a generic other return request reason
+- **Allow PickUp Points**: allow the customer to set a pick up point to drop off the items to return. It uses the geocoordinates from the order to find the closest pickup points.
+- **Proportional shipping value**: the shipping value to be refunded per item will be automatically calculated based on the item weight on the total order value. 
+
+### Transactional Emails
+The app leverages the capabilites of VTEX Message Center to notify the customers when a return request is created and when the status of their return changes. 
+
+The app creates a default template `oms-return-request-confirmation` that is modifiable on the Message Center to suit each store needs. 
```

Comment:
The app doesn't create it by default, only after sending the first email. It will be in English and the name will have the locale the store user was browsing the store. And current we have two templates. Besides `oms-return-request-confirmation_{locale}` we also have `oms-return-request-status-update_{locale}`.